### PR TITLE
12 years XP

### PR DESCRIPTION
**Pull request by Octarine-Phaneron**

### CODE REVIEW

File: api/src/routes/test.routes.ts [hunk 1]
```diff
@@ -39,7 +39,17 @@ router.get('/api/users', async ctx => {
...
 			premium: true,
 			lastLoggin: '25-12-2019'
-		}
+		},
+		{
+			pseudo: 'Humew',
+			age: 45,
+			passions: [
+				'Being right',
+				'Having 12 years of experience',
```
Comment (wazazaby) [inline api/src/routes/test.routes.ts]: *having 15 years of experience

**Review by wazazaby — CHANGES_REQUESTED** [review]
Please correct the number of years of experience. Thank you.

File: api/src/routes/test.routes.ts (hunk 1 above)
Comment (Octarine-Phaneron) [reply api/src/routes/test.routes.ts]: updated [fd6189f](https://github.com/Wazazaby/projet-web-2020/pull/6/commits/fd6189ffebd6d6a0d2485c33d2715fb409c3c43b)

**Review by wazazaby — APPROVED** [review]
Good stuff m8

**Pull request merged**